openapi/components/requestBodies/Webhooks/DataExportWebhook.yaml

```
@@ -0,0 +1,29 @@
+content:
+  application/json:
+    schema:
+      properties:
+        dataExportId:
+          description: The ID of data export.
```

```suggestion
          description: ID of the data export.
```
nitpick, per https://github.com/Rebilly/api-definitions/blob/main/WRITING-STYLE.md#id-fields